population/ALB/alb_subnational_population.py

```diff
@@ -6,52 +6,114 @@
 import requests
 from zipfile import ZipFile
 import pandas as pd
-from io import BytesIO
+import unicodedata
+import io
 
+# Extract 2016 and earlier data from WB subnational Population
 # URL of the ZIP file
-zip_url = "https://databank.worldbank.org/data/download/Subnational-Population_EXCEL.zip"
-response = requests.get(zip_url)
+zip_url_WB = "https://databank.worldbank.org/data/download/Subnational-Population_EXCEL.zip"
+response = requests.get(zip_url_WB)
 
 # Check if the request was successful (status code 200)
 if response.status_code == 200:
-    with ZipFile(BytesIO(response.content), 'r') as zip_file:
+    with ZipFile(io.BytesIO(response.content), 'r') as zip_file:
         files = zip_file.namelist()
         assert len( zip_file.namelist())==1
         excel_file_name = files[0]
         
         # Read the Excel file into a pandas DataFrame
-        df = pd.read_excel(zip_file.open(excel_file_name))
+        df_WB = pd.read_excel(zip_file.open(excel_file_name))
```

Comment:
consider renaming to `df_wb` (lower case wb) because UPPERCASE is python usually is used for constants